bundles/org.openhab.binding.chromecast/README.md

```diff
@@ -39,53 +39,53 @@ discovery.chromecast:background=false
+| episodeNumber   | Number      | All   | R   | (TvShowMediaMetadata) Episode number (in the season) of the t.v. show                                                                                                                 |
+| image           | Image       | All   | R   | (GenericMediaMetadata) Image for current media                                                                                                                                        |
+| imageSrc        | String      | All   | R   | (GenericMediaMetadata) URL of image for current media                                                                                                                                 |
+| locationName    | String      | All   | R   | (PhotoMediaMetadata) Verbal location where the photograph was taken; for example, "Madrid, Spain."                                                                                    |
+| location        | Location    | All   | R   | (PhotoMediaMetadata) Geographical location of where the photograph was taken                                                                                                          |
+| releaseDate     | DateTime    | All   | R   | (GenericMediaMetadata) ISO 8601 date and time this content was released                                                                                                               |
+| seasonNumber    | Number      | All   | R   | (TvShowMediaMetadata) Season number of the t.v. show                                                                                                                                  |
+| seriesTitle     | String      | All   | R   | (TvShowMediaMetadata) Descriptive title of the t.v. series                                                                                                                            |
+| studio          | String      | All   | R   | (TvShowMediaMetadata) Studio which released the content                                                                                                                               |
+| subtitle        | String      | All   | R   | (GenericMediaMetadata) Descriptive subtitle of the content                                                                                                                            |
+| title           | String      | All   | R   | (GenericMediaMetadata) Descriptive title of the content                                                                                                                               |
+| trackNumber     | Number      | All   | R   | (MusicTrackMediaMetadata) Number of the track on the album                                                                                                                            |
```

Comment:
Why is it necessarry to have a column where all rows have the same value ("All")?

Reply:
First i started having this column because i thought there where differences between the things. In the end it was not. In the  near future this might change, and i thought it was a good idea to have it explicitly set. Let me know if i have to remove it or not.

Reply:
I would suggest to remove it until there is a need for it.

Reply:
I would vote for removing it too, as long as it is not needed, thinking especially about users viewing those contents on a phone/tablet.